### PR TITLE
Solves NQP-148 (Reduced Effectiveness of Effects on Multi-tile Entities)

### PR DESCRIPTION
**Pull request by maxilevi**

### CODE REVIEW

**Comment by Snayff** [comment]
Lint and Test failed on build. Can you fix and resubmit the PR, please?

**Comment by maxilevi** [comment]
> Lint and Test failed on build. Can you fix and resubmit the PR, please?

I know, that is why it's a draft still.

**Comment by Snayff** [comment]
Ah, sorry!

File: data/config/game.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "default_values": {
         "__dataclass__": "DefaultValueData",
         "time_per_round": 20,
-        "entity_blocks_sight": true
+        "entity_blocks_sight": true,
+        "reduced_effectiveness_multi_tile_modifier": 0.25
```
Comment (maxilevi) [inline data/config/game.json:31]: Let me know if you can think of a more descriptive name

File: scripts/nqp/actions/skills.py [hunk 2]
```diff
@@ -129,17 +129,20 @@ def apply(self) -> Iterator[Tuple[EntityID, List[Effect]]]:
         """
         An iterator over pairs of (affected entity, [effects])
         """
-        entity_names = []
+        applied_entities: collections.defaultdict = collections.defaultdict(lambda: 1)
```
Comment (maxilevi) [inline scripts/nqp/actions/skills.py:132]: collections.defaultdict is like a normal dict but has a specified default value for every key.

**Pull request merged**